core/examples/luigi-example-angular/package.json

```diff
@@ -36,6 +36,7 @@
     "@types/node": "8.10.56",
     "glob-parent": "6.0.2",
     "lodash": "4.17.21",
+    "rollup": "4.22.5",
```

Comment:
```suggestion
```
I don't think you need this. rollup here was a child dependency of `@angular-devkit/build-angular`, which you already update to latest version above. That contains the updated rollup inside.